packages/firestore/src/lite/user_data_reader.ts

```diff
@@ -73,8 +73,8 @@ const RESERVED_FIELD_REGEX = /^__.*__$/;
  * lite, firestore-exp and classic SDK.
  */
 export interface UntypedFirestoreDataConverter<T> {
```

Comment:
Please also update the docs for FirestoreDataConverter to explain what this new input type is.